fiddler/pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools<61",
+    "fiddler-client==1.0.6",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-fiddler"
+description = "The fiddler check"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "fiddler",
+]
+authors = [
+    { name = "Rajeev", email = "rajeev@fiddler.ai" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 2.7",
```

Comment:
same question as below

Do you plan to support python 2.7? I don't see this version in the tox file

Reply:
@FlorentClarret - there was a comment earlier in an email thread from Alex of using an older version of python (2.7). We were thinking that some of the errors might be because of that and hence I explicitly removed 2.7. I would like to get this working first with 3.8 and then later add support for 2.7. Will keep that in mind to add it later on.

Reply:
I understand. In that case you can remove Python 2.7 from this section in this case and add it later if needed